Reference/google/protobuf/any.pb.swift

```diff
@@ -147,38 +155,52 @@ struct Google_Protobuf_Any: SwiftProtobuf.Message, SwiftProtobuf._MessageImpleme
   ///  
   ///   Schemes other than `http`, `https` (or the empty scheme) might be
   ///   used with implementation specific semantics.
-  var typeURL: String = ""
+  var typeURL: String {
+    get {return _storage._typeURL}
+    set {_uniqueStorage()._typeURL = newValue}
+  }
 
   ///   Must be a valid serialized protocol buffer of the above specified type.
-  var value: Data = Data()
+  var value: Data {
+    get {return _storage._value}
+    set {_uniqueStorage()._value = newValue}
+  }
 
   var unknownFields = SwiftProtobuf.UnknownStorage()
 
   init() {}
 
   mutating func decodeMessage<D: SwiftProtobuf.Decoder>(decoder: inout D) throws {
-    while let fieldNumber = try decoder.nextFieldNumber() {
-      switch fieldNumber {
-      case 1: try decoder.decodeSingularStringField(value: &typeURL)
-      case 2: try decoder.decodeSingularBytesField(value: &value)
-      default: break
+    _ = _uniqueStorage()
+    try withExtendedLifetime(_storage) { (_storage: AnyMessageStorage) in
+      while let fieldNumber = try decoder.nextFieldNumber() {
+        switch fieldNumber {
+        case 1: try decoder.decodeSingularStringField(value: &_storage._typeURL)
+        case 2: try decoder.decodeSingularBytesField(value: &_storage._value)
+        default: break
+        }
       }
     }
   }
 
   func traverse<V: SwiftProtobuf.Visitor>(visitor: inout V) throws {
-    if !typeURL.isEmpty {
-      try visitor.visitSingularStringField(value: typeURL, fieldNumber: 1)
-    }
-    if !value.isEmpty {
-      try visitor.visitSingularBytesField(value: value, fieldNumber: 2)
+    try withExtendedLifetime(_storage) { (_storage: AnyMessageStorage) in
+      try _storage.preTraverse()
+      if !_storage._typeURL.isEmpty {
+        try visitor.visitSingularStringField(value: _storage._typeURL, fieldNumber: 1)
+      }
+      if !_storage._value.isEmpty {
+        try visitor.visitSingularBytesField(value: _storage._value, fieldNumber: 2)
+      }
     }
     try unknownFields.traverse(visitor: &visitor)
   }
 
   func _protobuf_generated_isEqualTo(other: Google_Protobuf_Any) -> Bool {
-    if typeURL != other.typeURL {return false}
-    if value != other.value {return false}
+    if _storage !== other._storage {
```

Comment:
<s>Why not just `if _storage !== other._storage && !_storage.isEqualTo(other: other._storage) {return false}`?</s> Never mind, now I see the different code paths that the generator can take that produced this.

Shouldn't the other generated protos with storage change slightly as well because of the generator change?